subprojects/wrapper/src/main/java/org/gradle/wrapper/Download.java

```diff
@@ -52,13 +52,9 @@ public Download(Logger logger, DownloadProgressListener progressListener, String
         this.appName = appName;
         this.appVersion = appVersion;
         this.progressListener = new DefaultDownloadProgressListener(logger, progressListener);
-        configureProxyAuthentication();
-    }
 
-    private void configureProxyAuthentication() {
-        if (System.getProperty("http.proxyUser") != null) {
-            Authenticator.setDefault(new ProxyAuthenticator());
-        }
+        // Only an authenticator for proxies needs to be set. Basic authentication is supported by directly setting the request header field.
+        Authenticator.setDefault(new ProxyAuthenticator());
```

Comment:
Just to be on the safe side, I'd set the global authenticator if necessary.
```suggestion
       if (System.getProperty("http.proxyUser") != null || System.getProperty("https.proxyUser") != null) {
            Authenticator.setDefault(new ProxyAuthenticator());
        }```